src/domains/user/service.ts

```diff
@@ -1,5 +1,18 @@
 import { api } from "@/services/api";
+import { CreateUserData } from "../auth";
 
 export function getMe() {
-  return api.get("me");
+  return api.get("auth/me");
+}
+
+export async function updateUser({
+  id,
+  data,
+}: {
+  id: string;
+  data: Partial<CreateUserData>;
```

Comment:
I suggest you to have a separate type: `UpdateUserData`

```ts
export type UpdateUserData = Partial<CreateUserData>
```

Also, consider removing password from the update data. Refer to my comments on backend project.

```ts
export type UpdateUserData = Omit<Partial<CreateUserData>, 'password'>
```